src/modules/kvs/commit.c

```diff
@@ -163,6 +163,13 @@ int commit_set_aux_errnum (commit_t *c, int errnum)
     return c->aux_errnum;
```

Comment:
Suggestion: modules/kvs: try commits individually if merged commit fails

"core KVS file" could be KVS main or kvs.c.

Reply:
or "try orig commit if merged comit fails"